assets/js/modules/analytics/settings/settings-edit.js

```diff
@@ -47,6 +48,7 @@ export default function SettingsEdit() {
 	const isDoingSubmitChanges = useSelect( ( select ) => select( STORE_NAME ).isDoingSubmitChanges() );
 	const haveSettingsChanged = useSelect( ( select ) => select( STORE_NAME ).haveSettingsChanged() );
 	const isCreateAccount = ACCOUNT_CREATE === accountID;
+	const usingProxy = useSelect( ( select ) => select( 'core/site' ).isUsingProxy() );
```

Comment:
This should be importing `STORE_NAME` from the core site store rather than hard coding it.
Otherwise, this is beautiful!

Reply:
👍 Will do!

Reply:
Done.